config/config.exs

```diff
@@ -1,5 +1,7 @@
 import Config
 
+config :logger, backends: []
```

Comment:
Isn't it just a leftover?

Reply:
No, unfortunately the logger does not work properly without this line